### PR TITLE
[v7r2] Move DIRAC/VOPolicy/InputDataModule under Operations/InputDataPolicy

### PR DESCRIPTION
**Pull request by zmathe**
BEGINRELEASENOTES

*Interfaces
CHANGE: Move DIRAC/VOPolicy/InputDataModule to DIRAC/Operations/InputDataPolicy

ENDRELEASENOTES

### CODE REVIEW

**Comment by fstagni** [comment]
Why do you want to move the location?

**Comment by zmathe** [comment]
There are two issues:
-it is not multi-vo supports
-It assumes that you have a proxy. However it can be a host certificate. In addition the same check is already in https://github.com/DIRACGrid/DIRAC/blob/integration/ConfigurationSystem/Client/Helpers/Operations.py#L111
So, I just simplify the code and make it better and in addition multi vo also supported.

**Comment by fstagni** [comment]
Add to the wiki.

**Comment by fstagni** [comment]
Please rebase.

**Comment by fstagni** [comment]
Please rebase, one last time. And start the v7r2 wiki in https://github.com/DIRACGrid/DIRAC/wiki with this change.

**Comment by zmathe** [comment]
@fstagni I rebased and created the wiki. I do not know why the CI is failing.

File: Interfaces/API/Dirac.py [hunk 1]
```diff
@@ -45,11 +45,13 @@
 from DIRAC.Core.Utilities.ClassAd.ClassAdLight import ClassAd
 from DIRAC.Core.Utilities.Subprocess import systemCall
 from DIRAC.Core.Utilities.ModuleFactory import ModuleFactory
+from Core.Utilities.Decorators import deprecated
```
Comment (fstagni) [inline Interfaces/API/Dirac.py]: this import is wrong

**Comment by fstagni** [comment]
Please rebase.

**Comment by fstagni** [comment]
Please rebase one last time.

**Comment by zmathe** [comment]
@fstagni Done

File: Interfaces/API/Dirac.py [hunk 2]
```diff
@@ -591,6 +593,7 @@ def __monitorSubmittedJob(self, jobID):
 
   #############################################################################
   @staticmethod
+  @deprecated("It will be removed in v7r2. Use Operations sections only!")
```
Comment (fstagni) [inline Interfaces/API/Dirac.py:596]: I guess you meant
```suggestion
  @deprecated("It will be removed after v7r2. Use Operations sections only!")
```

**Pull request merged**